content/docs/reference/providers/pulumi/aws/s3/bucket.md

```diff
@@ -0,0 +1,1827 @@
+        <tr>
+            <td class="align-top">website<wbr>Endpoint</td>
+            <td class="align-top"><code>string</code></td>
+            <td class="align-top">{{% md %}}
+The website endpoint, if the bucket is configured with a website. If not, this will be an empty string.
+
+{{% /md %}}</td>
+        </tr>
+    </tbody>
+</table>
+
+## Look up an Existing Bucket Resource
+
+{{< langchoose csharp >}}
+
+```typescript
+public static get(name: string, id: pulumi.Input<pulumi.ID>, state?: BucketState, opts?: pulumi.CustomResourceOptions): Bucket;
+```
+
+```python
+def get(resource_name, id, opts=None, acceleration_status=None, acl=None, arn=None, bucket=None, bucket_domain_name=None, bucket_prefix=None, bucket_regional_domain_name=None, cors_rules=None, force_destroy=None, hosted_zone_id=None, lifecycle_rules=None, loggings=None, object_lock_configuration=None, policy=None, region=None, replication_configuration=None, request_payer=None, server_side_encryption_configuration=None, tags=None, versioning=None, website=None, website_domain=None, website_endpoint=None)
+```
+
+```go
+func GetBucket(ctx *pulumi.Context, name string, id pulumi.IDInput, state *BucketState, opts ...pulumi.ResourceOption) (*Bucket, error)
+```
+
+```csharp
+public static Bucket Get(string name, Input<string> id, BucketState? state = null, CustomResourceOptions? options = null);
+```
+
+Get an existing Bucket resource's state with the given name, ID, and optional extra
+properties used to qualify the lookup.
+
+{{% lang nodejs %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>id</strong> &ndash; (Required) The _unique_ provider ID of the resource to lookup.</li>
+    <li><strong>state</strong> &ndash; (Optional) Any extra arguments used during the lookup.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+{{% lang go %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>id</strong> &ndash; (Required) The _unique_ provider ID of the resource to lookup.</li>
+    <li><strong>state</strong> &ndash; (Optional) Any extra arguments used during the lookup.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+{{% lang csharp %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>id</strong> &ndash; (Required) The _unique_ provider ID of the resource to lookup.</li>
+    <li><strong>state</strong> &ndash; (Optional) Any extra arguments used during the lookup.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+The following state arguments are supported:
```

Comment:
I'll admit, it bums me out that this uses up so much space when it's actually so rare to use. 90% of the time, you're just using the ID. Maybe we can link to the input arguments above and say it's option?